src/lib/filter-button-author.svelte

```diff
@@ -0,0 +1,84 @@
+<script>
+    import AuthorsFilter from "$lib/authors-filter.svelte";
+    import PaginatedView from "./paginated-view.svelte";
+
+    /** @type {import('./$types').PageData} */
+    let { author = $bindable(), authors, totalAuthors, authorsPage } = $props();
+    let isDropdownOpen = $state(false);
+
+    function closeDropdown() {
+        isDropdownOpen = false;
+    }
+</script>
+<div>
+    <input type="checkbox" id="author-filter-more" bind:checked={isDropdownOpen}/>
+    <label for="author-filter-more" class="author-filter-more-label" aria-label="More filters">More...</label>
+    <div class="card card-compact z-[1] w-64 p-2 ">
+        <div class="card-body">
+            <div class="filter-header">
+                <PaginatedView name="authors" bind:pageNr={authorsPage} totalResults={totalAuthors} perPage=66/>
+                <label for="author-filter-more" class="btn" aria-label="Close popup">
+                    <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        fill="none"
+                        viewBox="0 0 24 24"
+                        class="inline-block h-4 w-4 stroke-current">
+                        <path
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
+                        stroke-width="2"
+                        d="M6 18L18 6M6 6l12 12"></path>
+                    </svg>
+                </label>
+            </div>
+
+            <AuthorsFilter bind:author={author} {authors} {totalAuthors} {authorsPage} hideFn={closeDropdown}/>
+        </div>
+    </div>
+</div>
+<style>
```

Comment:
There is a considerable amount of work done in styling these filters. Is this the final design? If not, it might not be worth the effort and time, and use out of the box elements from the component lib/design system instead.

For example dropping in this component would probably achieve the same thing: https://v4.daisyui.com/components/menu/#submenu